vuepress/docs/v6.3.2/docs/getting-started/README.md

```diff
@@ -4,17 +4,43 @@ sidebarDepth: 2
 
 # Getting Started with Entando
 
-::: warning
+::: warning 
 **Mac:** Entando 6 is not currently compatible with the Apple M1 ARM64 architecture found in some newer Macs.
 :::
 
-There are two options for getting started with the Entando platform.
-1. [Automatically install Entando via the Entando command-line interface (CLI)](#automatic-install). This is the fastest way to initialize an Entando application in Kubernetes. A CLI install defaults certain settings and prepares a local developer environment.
-2. [Manually install Entando step by step](#manual-install). This permits configuration of either a shared cluster or a local developer environment. A manual install is appropriate when CLI defaults don't meet specific needs, or to customize the deployment.
+## Try Entando
+
+Set up Entando using two simple steps or install manually for your specific needs. Then try the platform with a simple widget or walk through the **Welcome Wizard** in our AppBuilder. For a more in-depth experience, check out our **Banking demo**.
```

Comment:
We should probably update the language to Standard Banking Demo rather than Banking demo or Standard Demo. That's what the tutorial is called now.

Reply:
Glossary - App Builder, instead of AppBuilder.

Reply:
i think it's important to keep 1. and 2. above. they provide pertinent information.

Reply:
never mind! i just got to where you moved that info to. it flows nicely to introduce things vaguely then hit the details time-of. on the other hand 1. and 2. clue the user into which install they'd like to follow and quickly compare the two... me not knowing anything i'd like the info up front, but someone with experience would probably know which path to take and then get a refresh on the particulars :)

Reply:
done